mempool/cat/pool.go

```diff
@@ -0,0 +1,765 @@
+package cat
+
+import (
+	"errors"
+	"fmt"
+	"runtime"
+	"sort"
+	"sync"
+	"time"
+
+	"github.com/creachadair/taskgroup"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tendermint/libs/log"
+	"github.com/tendermint/tendermint/mempool"
+	"github.com/tendermint/tendermint/proxy"
+	"github.com/tendermint/tendermint/types"
+)
+
+var _ mempool.Mempool = (*TxPool)(nil)
+
+const evictedTxCacheSize = 200
```

Comment:
[non-blocking] was there any decision making behind this number? Or, what are the considerations that should go into this number if we think to change it in the future?  A comment would be great.